File: 2st/Riyeon/마지막 두 원소.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[num_list.length+1];
        for(int i=0; i<num_list.length; i++){
            answer[i] = num_list[i];
            if(i==num_list.length-1)
                answer[i+1] = num_list[i] > num_list[i-1]? num_list[i]-num_list[i-1]:num_list[i]*2;
        }
        return answer;
    }
}

File: 2st/Riyeon/마지막 두 원소.py
def solution(num_list):
    i = len(num_list) -1
    if num_list[i] > num_list[i-1]:
        num_list.append(num_list[i] - num_list[i-1])
    else:
        num_list.append(num_list[i] *2)
    return num_list